Project.toml

```diff
@@ -6,6 +6,7 @@ version = "0.12.1-DEV"
 [deps]
 AbstractAlgebra = "c3fe647b-3220-5bb0-a1ea-a7954cac585d"
 AlgebraicSolving = "66b61cbe-0446-4d5d-9090-1ff510639f9d"
+Distributed = "8ba89e20-285c-5b6f-9357-94700520ee1b"
```

Comment:
```suggestion

```
This should not be needed here, only in `test/Project.toml`.

Reply:
Indeed. Also, this unfortunately now needs to be rebased again